### ISSUE TITLE
--debug and --debug-brk doesn't work on node 8.x: Add support for --inspect / --inspect-brk

### ISSUE THREAD

**Issue by aneilbaboo** [mochajs/mocha#2925]
Node 8.x deprecates the --debug and --debug-brk flags, and fails with the following message:

```shell
$ nvm use 8.1.4
Now using node v8.1.4 (npm v5.0.3)
$ mocha --debug-brk                                                                  
(node:18978) [DEP0062] DeprecationWarning: node --debug and node --debug-brk are invalid. 
```

Currently, mocha ignores `--inspect` and `--inspect-brk`.  

Can we pass these through?

**Comment by aneilbaboo** [comment]
Ignore - this is fixed in 3.4.2

**Comment by Waog** [comment]
got the same issue with mocha `3.5.3`:

```
mocha --debug --require ts-node/register "test/unit/**/*.test.ts" "src/**/*.test.ts"

(node:12940) [DEP0062] DeprecationWarning: `node --debug` and `node --debug-brk` are invalid. Please use `node --inspect` or
`node --inspect-brk` instead.
npm ERR! code ELIFECYCLE
npm ERR! errno 9
npm ERR! @vestr/core@0.0.1 test:unit:debug: `mocha --debug --require ts-node/register "test/unit/**/*.test.ts" "src/**/*.test.ts"`
npm ERR! Exit status 9
npm ERR!
npm ERR! Failed at the @vestr/core@0.0.1 test:unit:debug script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Oliver\AppData\Roaming\npm-cache\_logs\2018-05-15T13_19_14_459Z-debug.log
```
